add_portal_usermenu_item/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Add Portal menu item in Usermenu",
+    "summary": "Display portal menu item under the usermenu list.",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Portal",
+    "version": "15.0.1.0.0",
+    "depends": ["web"],
+    "assets": {
+        "web.assets_backend": [
+            "add_portal_usermenu_item/static/src/**/*.esm.js",
```

Comment:
What does 'esm' stand for?

I prefer to make the path explicit (instead of using wildcard strings like `**/*`) when there is only one file.

Reply:
If I don't use esm , I got error when I run pre-commit because the .eslintrc config file only allow for esm.js. If I don't want to use esm, we need to add "source:module" under parserOptions in .eslintrc.

Reply:
OK, got it. Found the background here: https://github.com/OCA/oca-addons-repo-template/issues/85#issuecomment-952881893